classes/task/ReviewReminder.inc.php

```diff
@@ -39,14 +45,14 @@ public function getName()
 
     /**
      * Send the automatic review reminder to the reviewer.
-     *
-     * @param \PKP\submission\reviewAssignment\ReviewAssignment $reviewAssignment
-     * @param Submission $submission
-     * @param Context $context
-     * @param string $reminderType
-     * 	REVIEW_REMIND_AUTO, REVIEW_RESPONSE_OVERDUE_AUTO
+     * @return bool|void
```

Comment:
can be type hinted on the function as `?bool`

Reply:
It requires deliberately changing the return type in the method's body.